zilencer/models.py

```diff
@@ -284,6 +294,16 @@ class Meta:
                 fields=["property", "end_time"],
                 name="zilencer_remoterealmcount_property_end_time_506a0b38_idx",
             ),
+            models.Index(
+                fields=["server", "realm_id"],
+                condition=Q(remote_realm__isnull=True),
+                name="zilencer_remoterealmcount_server_realm",
+            ),
+            models.Index(
+                fields=["server"],
+                condition=Q(remote_realm__isnull=True),
+                name="zilencer_remoterealmcount_server",
+            ),
```

Comment:
Do these indexes look right? The `server` one is for the `if not RemoteRealmCount.objects.filter(server=server, remote_realm=None).exists():` query. Or is that index redundant given there's an ordinary `server` index, since it's a `ForeignKey`?

The `(server, realm_id)` one is for
```
        RemoteRealmCount.objects.filter(server=server, remote_realm=None, realm_id=realm_id).update(
            remote_realm=realm_id_to_remote_realm[realm_id]
        )
```

Reply:
Indexes with a condition that should become increasingly rare are pretty cheap, so I think we can run with this.

We may want to later add some indexes on `server, remote_realm, ...` for future code that might query it ... but let's discuss that in chat.